src/measure_theory/integral/circle_integral_transform.lean

```diff
@@ -0,0 +1,155 @@
+/-
+Copyright (c) 2022 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+import analysis.complex.cauchy_integral
+import analysis.analytic.basic
+import analysis.calculus.parametric_interval_integral
+import data.complex.basic
+import measure_theory.integral.circle_integral
+/-!
+# Circle integral transform
+
+In this file we define the circle integral transform of a function `f` with complex domain. This is
+defined as $(2πi)^{-1}\frac{f(x)}{x-w}$ where `x` moves along a circle. We then prove some basic
+facts about these functions.
+
+These results are useful for proving that the uniform limit of a sequence of holomorphic functions
+is holomorphic.
+
+-/
+
+open set measure_theory metric filter function
+open_locale interval real
+
+noncomputable theory
+
+variables {E : Type} [normed_group E] [normed_space ℂ E]
```

Comment:
How about:
```suggestion
variables {E : Type} [normed_group E] [normed_space ℂ E] (R : ℝ) (z w : ℂ)
```
since almost every `def` / `lemma` below uses these?